backend/app/api/endpoints/wiki.py

```
@@ -217,6 +218,35 @@ def save_wiki_generation_contents(
     return None
 
 
+@internal_router.get("/generations/{generation_id}/pages", response_model=WikiPageRead)
+def read_wiki_generation_page(
+    generation_id: int,
+    path: str = Query(..., min_length=1, description="Stable page path to read"),
+    _: None = Depends(_verify_internal_token),
+    wiki_db: Session = Depends(get_wiki_db),
```

Ruff (0.16.0)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)